src/matcha_ml/cli/cli.py

```diff
@@ -81,11 +82,24 @@ def get(
         show_sensitive (Optional[bool]): show hidden sensitive resource values when True. Defaults to False.
 
     Raises:
-        Exit: Exit if matcha remote state has not been provisioned.
         Exit: Exit if matcha.state file does not exist.
+        Exit: Exit if matcha remote state has not been provisioned.
         Exit: Exit if resource type or property does not exist in matcha.state.
     """
+    matcha_state_path = os.path.join(".matcha", "infrastructure", "matcha.state")
```

Comment:
There's a separate ticket RPD-235 to separate all this logic into `core.py` for get command.

Reply:
Okay - my main feedback here would have been that this logic should be in the core, but I suppose it can be done in RPD-235. I think I'd prefer this logic to be in core already, prior to the refactor (seeing as it's new)

Reply:
I had to refactor `RemoteStateManager`  inside `core.py`  because the mocking gets difficult if I have to mock two `RemoteStateManager` one inside `cli.py`  and another inside `core.py`  for `get`  command.